### PR TITLE
add static site infrastructure

### LINKED ISSUE #1: Create static site and host on github pages
Use a static site generator to deploy a simple homepage and enable a blog to be statically generated and hosted at a url like `docudump.github.io/blog`

Due date: 2024-02-01

### PR DESCRIPTION
**Pull request by collinb424**
- Used this Hugo template: https://github.com/zeon-studio/hugoplate?tab=readme-ov-file
    - The repo has good instructions on how to get set up. Note that you need Hugo Extended v0.115+, Node v18+, and Go v1.20+
    - For getting Hugo, I'd recommend downloading it manually from here so you get the latest version (be sure you get the extended versions down at the bottom): https://github.com/gohugoio/hugo/releases/tag/v0.122.0
- I kept most of the content from the template and only changed a couple things. We can decide on layout, pictures, logo, content, etc later. I'd recommend following the instructions in the above repo to get it running locally and get a feel for how making changes works. Changing shouldn't be much work; we can make basically all the changes in markdown.

### CODE REVIEW

**Comment by dblitt** [comment]
Thanks for taking care of this! A few notes from the top of my head:

1. It looks like there are tons of project files we don't need (like gitlab-ci.yml, netlify.toml, old readme.md, etc). Those can be removed. Also see `.github/ISSUE_TEMPLATE/bug_report.yml`
2. `LICENSE` should be updated to include us (DocuDump Contributors)
3. This puts in a ton of example pages and images. Can they be reduced to maybe one of each type? We don't need all this bloat.
4. The name and description in `package.json` should be updated to be relevant to this project
5. We should make another issue to remove all the boilerplate text from the website and replace it with relevant text, but that can be outside the scope of this PR.

**Review by dblitt — CHANGES_REQUESTED** [review]
See previous comment.

File: content/english/_index.md [hunk 1]
```diff
@@ -28,28 +28,28 @@ features:
...
-    content: "Hugo is an all-in-one web framework for building fast, content-focused websites. It offers a range of exciting features for developers and website creators. Some of the key features are:"
-    bulletpoints:
-      - "Zero JS, by default: No JavaScript runtime overhead to slow you down."
-      - "Customizable: Tailwind, MDX, and 100+ other integrations to choose from."
-      - "UI-agnostic: Supports React, Preact, Svelte, Vue, Solid, Lit and more."
-    button:
-      enable: true
-      label: "Get Started Now"
-      link: "https://github.com/zeon-studio/hugoplate"
+  # - title: "Discover the Key Features Of Hugo"
```
Comment (dblitt) [inline content/english/_index.md]: Don't bother commenting, just delete

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "hugoplate",
-  "description": "hugo tailwindcss boilerplate",
+  "name": "DocuDump",
```
Comment (dblitt) [inline package.json]: should be `docudump.github.io`

**Review by dblitt — CHANGES_REQUESTED** [review]
If you make these changes we can merge and verify the site builds.

But make an issue to remove the boilerplate pages (like keeping only one of each type) and assign it to yourself to get to later.

**Comment by kpconnors** [comment]
Took out a bunch of unnecessary items. Figured we can always look at the original if we want to reimplement a page but this static site is simply a landing page to download our app. 

Notes: 
- I left the home page and the contact page just so we can easily see how to add an additional pages. Updated header and footers accordingly.
- Kept the 404 page, this implementation works nicely.
- When we pick out a logo, we should name it logo.png and add it to the images folder. It would fit in various places on the site
- Took out some plugins(google maps API and swiper for mobile apps)

**Review by dblitt — APPROVED** [review]

**Pull request merged**